cli/templatedelete.go

```diff
@@ -57,17 +67,19 @@ func templateDelete() *cobra.Command {
 				for _, template := range allTemplates {
 					if template.Name == selection {
 						templates = append(templates, template)
+						templateNames = append(templateNames, template.Name)
 					}
 				}
 			}
 
-			for _, templateName := range templateNames {
-				template, err := client.TemplateByName(ctx, organization.ID, templateName)
-				if err != nil {
-					return xerrors.Errorf("get template by name: %w", err)
-				}
-
-				templates = append(templates, template)
+			// Confirm deletion of the template.
+			_, err = cliui.Prompt(cmd, cliui.PromptOptions{
+				Text:      fmt.Sprintf("Delete these templates: %s?", cliui.Styles.Code.Render(strings.Join(templateNames, ", "))),
+				IsConfirm: true,
+				Default:   "no",
```

Comment:
All of our other confirmation prompts default to "yes". We should probably make workspaces default to "no" as well.

Reply:
I think dangerous actions should default to safety.  All yes or all no isn't the appropriate framework; safe versus dangerous is.

Reply:
I think it's fine to default to safety here, I'm just promoting consistency among our destructive actions.

Reply:
I agree, but the way I want to manifest that agreement is changing the defaults on the others 🙃.  Given the existence of `--yes`, I think it shouldn't get in the way of the #yolo customers.

Reply:
#2878